tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/Checks/IisCheckTests.cs

```diff
@@ -230,7 +230,7 @@ private async Task<IisFixture> StartIis(IisAppType appType)
 
         try
         {
-            await fixture.TryStartIis(this, appType);
+            await fixture.TryStartIis(this, appType, false);
```

Comment:
This class has a more relaxed httpget below that allows forbidden status codes to pass the check.